### PR TITLE
[Docs] PR for Add flowchart for how to upgrade YBA when HA enabled

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6550

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:134]: fixed

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:138]: changed

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:142]: Here's what's in the google doc provided by dev: 

_7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade._

And here's the corresponding step in the diagram:

_Promote Failover with latest Version B backup._

I'm changing step 7 to:

_7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade._

Waiting for dev to review and let me know what it should be.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:144]: I know. The dev google doc has it as a step and it's an element on the diagram. If I don't give it a number, it'll break the description sequence.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:146]: Yes, the whole thing needs work. I'm waiting for dev comments.

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-kubernetes.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-kubernetes.md:38]: Changed

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md:46]: Changed

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-kubernetes.md
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-kubernetes.md:38]: If you are upgrading

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md:46]: If you are upgrading

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:154]: Sentence not complete?

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-kubernetes.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-kubernetes.md:38]: fixed

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md:46]: fixed

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:154]: fixed

**Comment by lizayugabyte** [comment]
@sb-yb @eugeneckim Could you, please, review this doc PR and let me know how to address inconsistencies between the diagram and description from the google doc

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (eugeneckim) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:160]: @ddhodge / @lizayugabyte 

Your assessment is accurate, where the final line had a gap and should be heading back to ABOVE the Stop HA Sync.  I've updated the diagram, where the presence of the diamond is a limitation of the language used to create the diagram.  I didn't realize this document was going to be used as the basis for this request.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md]: @eugeneckim Could you, please, clarify if step 7 is correct as:

7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:146]: @eugeneckim Could you, please, review the steps and let me know if they've been captured correctly?

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md]: Replaced the diagram with the updated version.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (eugeneckim) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md]: > Don't quite follow - update the upgraded standby using a backup from the upgraded active?

Stated differently: This step is suggesting the user, as part of validation, can perform a test failover.  It's optional, as customers will have various exit criteria.  This specific customer wanted to test failovers as part of their upgrade validation.

> Here's what's in the google doc provided by dev:
> 
> _7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade._
> 
> And here's the corresponding step in the diagram:
> 
> _Promote Failover with latest Version B backup._

"Version B" was a shortened version of "YugabyteDB Anywhere version to which to upgrade" so that the text would fit into the diagram.  It was depicted as a note since it's an example subset of "Validate promotions."

> 
> I'm changing step 7 to:
> 
> _7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade._
> 
> Waiting for dev to review and let me know what it should be.

I compared the two versions of Step 7 in the comment and could not determine a difference between the two.

I went back to the original doc and the original step 7 said:

> Optionally perform promotion of Failover to with latest backup synchronized from Version B.

Comparing these two, I agree with the proposed change.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (eugeneckim) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:146]: The step 9 above as worded is accurate.

Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:146]: Steps are accurate, suggesting splitting into 2 sections however. First part ends at 7. Then 'If the upgrade failed, do the following:' and start from 1 again at step 9. Because 9 does not follow from 8 or 7.

Comment (eugeneckim) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:146]: @ddhodge I agree, the sections based on function/scenario does make things more clear.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md]: Okay, I'm keeping the modified step 7.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:146]: Changed the list formatting

**Review by eugeneckim — COMMENTED** [review]
These look OK to me. One outstanding item is if we want to include the diagram from the document as it's not consistent with other diagrams found in YB documentation.  I will defer to the documentation team on what is recommended here.

**Pull request merged**
